examples/src/main/java/com/mapbox/navigation/examples/MainActivity.kt

```diff
@@ -12,10 +12,8 @@ import androidx.core.content.ContextCompat
 import com.mapbox.android.core.permissions.PermissionsListener
 import com.mapbox.navigation.examples.settings.NavigationSettingsActivity
 import com.mapbox.navigation.navigator.internal.MapboxNativeNavigatorImpl
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
is the "wild card import" something we want?

Reply:
wild card import is applicable for kotlin android extensions only. I think it always was a case 🤔 I think we could restrict it 👍

Reply:
yeah, it's fine. I just see inconsistency so wondering if this is something we could archive by `ktlint format`?

for inconsistency, I mean like: https://github.com/mapbox/mapbox-navigation-android/pull/3513/files/3ac9b6e24d1ba42662b3a2bb72674e3cdeb86a44#diff-14b1d1bb5786020c720685dfdbbf5715L46-L50

Reply:
@JunDai seem, that we need an only one format for these imports. Thanks for feedback!

Reply:
I don't think it's a blocker of this PR, but it would be great if we find a way to keep it consistency.

Reply:
seems that we cannot control wildcard imports on our side for `kotlinx.android.synthetic` imports  https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/NoWildcardImportsRule.kt#L18 
We could write own rule for it but believe it should be another pr